internal/helper/sqlMutations.go

```diff
@@ -0,0 +1,111 @@
+package helper
```

Comment:
Let's name this file `sql_conversions.go` (underscore for consistency, "conversions" not tied to, but sounds clearer to me this is helpers for type conversions)